src/reducers/file-search.js

```diff
@@ -87,6 +96,15 @@ function update(
       return state.setIn(["modifiers", action.modifier], actionVal);
     }
 
+    case "NAVIGATE": {
+      return state.set("query", "").set("searchResults", {
+        matches: [],
+        matchIndex: -1,
+        index: -1,
+        count: 0
+      });
```

Comment:
are these the default params? maybe we can just pull that out into its own object...

` state.set("query", "").set("searchResults", emptySearchResults)`